javascript.js

```diff
@@ -131,3 +131,95 @@ function resetIngredientFields() {
     <input type="text" id="ingredient5" name="ingredient5" required><br><br>
   `;
 }
+let recipes = [
```

Comment:
You could add the recipe that the user created with the form implemented in week 1 to this array too. It would then become searchable and displayed in the grid of recipes.